netweaver/install_aas.sls

```diff
@@ -87,5 +87,6 @@ remove_aas_inifile_{{ instance_name }}:
     - name: {{ inifile }}
     - require:
       - create_aas_inifile_{{ instance_name }}
+      - netweaver_install_{{ instance_name }}
```

Comment:
I think we shouldn't put this `require`. We want to remove the configuration file always, not only when the installation works. The config file stores some secrets that we don't want to keep there.

Reply:
I was thinking incase of failure, the inifile maybe useful for debugging.
But yes, from security perspective, that makes sense.